### ISSUE TITLE
Broken link to Open Science Guide 

### ISSUE THREAD

**Issue by maneesha** [LibraryCarpentry/lc-git#168]
### What is the problem?

In the [git reference section](https://librarycarpentry.org/lc-git/reference.html), there is a broken link to [Open Science Guide](https://book.openingscience.org.s3-website-eu-west-1.amazonaws.com/)

### Location of problem (optional)

https://librarycarpentry.org/lc-git/reference.html

**Comment by alex-ball** [comment]
Thanks for raising this issue, @maneesha .

This is a weird one. When you do manage to get to the book, the URL shown in the address bar is the one we currently link to, but if you go to the link directly the server doesn't respond. It seems you have to be redirected there after first going to [book.openingscience.org](https://book.openingscience.org/).

So yes, the link needs to be changed and the title of the book isn't quite right either.

**Comment by alex-ball** [comment]
@tmillenaar, thanks very much for taking a look at this.

I think the issue with the link is that the domain does not support HTTPS. I should have specified the protocol as well.

Do you have any better luck going directly to <http://book.openingscience.org/>?

**Comment by tmillenaar** [comment]
Hey @alex-ball, thanks for the quick reply!

Indeed, the domain not supporting HTTPS was also what I encountered. If I go directly to http://book.openingscience.org/ I was unable to load at all initially when using firefox. In chrome I can access the book that way if I tell my browser to accept the risk. I updated it with a url that does support HTTPS and from there you can decide to either follow the link you posted through HTTP or go to somewhere else to get the book, like https://link.springer.com/book/10.1007/978-3-319-00026-8 that does support HTTPS.

**Comment by alex-ball** [comment]
Good point. It sound like you might be using "HTTPS-Only Mode" in Firefox; I see there's an equivalent setting ("Always use secure connection") in Chromium-based browsers. Anyone with that setting active will run into the same issue.

While the intermediate page is extra work, there's extra value to it too, and it avoids the issue. You've convinced me!

**Comment by alex-ball** [comment]
Resolved by #186.
